### PR TITLE
Merchant items

### PR DESCRIPTION
**Pull request by JBelta**

### CODE REVIEW

File: spec/features/merchant/edit_spec.rb [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+          expect(page).to have_link('Edit Item')
+          click_link 'Edit Item'
+        end
+        expect(current_path).to eq("/merchant/items/#{@pull_toy.id}/edit")
+
+        #fill_in "Name", with: ""
+        fill_in "Description", with: "Okay, pull toy"
+        #fill_in "Image", with: ""
+        #fill_in "Price", with: "two"
+        #fill_in "Inventory", with: 50
```
Comment (SageOfCode) [inline spec/features/merchant/edit_spec.rb:39]: Can these be deleted?

File: spec/features/merchant/edit_spec.rb [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+        within "#item-#{@pull_toy.id}" do
+          expect(page).to have_link('Edit Item')
+          click_link 'Edit Item'
+        end
+        expect(current_path).to eq("/merchant/items/#{@pull_toy.id}/edit")
+
+        fill_in "Name", with: ""
+        #fill_in "Description", with: "Great pull toy!"
+        #fill_in "Image", with: ""
```
Comment (SageOfCode) [inline spec/features/merchant/edit_spec.rb:61]: Same with these.

**Review by SageOfCode — APPROVED** [review]
May need to delete a couple commented out lines, otherwise looks ready to merge.

**Review by Eternal-Flame085 — APPROVED** [review]
Other than the commented out section looks good to me

**Review by araspaas — APPROVED** [review]
Looks good to merge

**Pull request merged**